jpf-sv-comp

```diff
@@ -36,18 +36,22 @@ trap "rm -rf $DIR" EXIT
 
 mkdir -p $DIR/target/classes
 
-/usr/lib/jvm/java-8-openjdk-amd64/bin/javac -g -cp $CLASSPATH_ADDED:$DIR/target/classes -d $DIR/target/classes "${BM[@]}"
+$JAVA_HOME/bin/javac -g -cp $CLASSPATH_ADDED:$DIR/target/classes -d $DIR/target/classes "${BM[@]}"
 
```

Comment:
please revert back to the old path for the Java, as the sv-comp setup can have a different home than what we want. Unless you export the java 8 path, at which point it doesn't really matter